app/models/category.rb

```diff
@@ -0,0 +1,8 @@
+class Category < ApplicationRecord
+
+  has_many :users
+
+  validates :name, presence: true
+  
+  validates :name, uniqueness: true
```

Comment:
Do we want uniqueness here? Would a user ever be able to be both a villager and an organizer? @bfpepper @annadolan